backend/src/waterfallhunter/core/github_release_backup_verification.py

```
@@ -0,0 +1,217 @@
+            candidate.is_file()
+            and not candidate.is_symlink()
+            and stat_result.st_uid == 0
+            and stat_result.st_mode & 0o022 == 0
+        ):
+            return str(candidate)
+    raise TrustedRemoteBackupVerificationError("GITHUB_CLI_UNAVAILABLE_OR_UNTRUSTED")
+
+
+def _gh_json(endpoint: str) -> dict[str, Any]:
+    try:
+        completed = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)